js/tidex2.js

```diff
@@ -0,0 +1,960 @@
+        //                 "money":"BTC",
+        //                 "stockPrec":8,
+        //                 "feePrec":8,
+        //                 "minAmount":"0.001"
+        //             }
+        //         ]
+        //     }
+        //
+        const result = [];
+        for (let i = 0; i < markets.length; i++) {
+            const market = markets[i];
+            const id = this.safeString (market, 'name');
+            const [ baseId, quoteId ] = id.split ('_');
+            const base = this.safeCurrencyCode (baseId);
+            const quote = this.safeCurrencyCode (quoteId);
+            const precision = {
+                'amount': undefined,
+                'price': this.safeInteger (market, 'moneyPrec'),
+            };
+            result.push ({
+                'id': id,
+                'symbol': base + '/' + quote,
+                'base': base,
+                'quote': quote,
+                'settle': undefined,
+                'baseId': baseId,
+                'quoteId': quoteId,
+                'settleId': undefined,
+                'type': 'spot',
+                'spot': true,
+                'margin': false,
+                'swap': false,
+                'future': false,
+                'option': false,
+                'active': undefined,
+                'contract': false,
+                'linear': undefined,
+                'inverse': undefined,
+                'taker': undefined,
+                'contractSize': undefined,
+                'expiry': undefined,
+                'expiryDatetime': undefined,
+                'strike': undefined,
+                'optionType': undefined,
+                'precision': precision,
+                'limits': {
+                    'leverage': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'amount': {
+                        'min': this.safeNumber (market, 'minAmount'),
+                        'max': undefined,
+                    },
+                    'price': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'cost': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                },
+                'info': market,
+            });
+        }
+        return result;
+    }
+
+    parseTicker (ticker, market = undefined) {
+        //
+        // fetchTicker
+        //     {
+        //         "name":"YFI_USDT",
+        //         "bid":"21607.842223",
+        //         "ask":"21635.50514299",
+        //         "open":"21530.31",
+        //         "high":"22619.38",
+        //         "low":"21193.59",
+        //         "last":"21644.23",
+        //         "volume":"188.4555124",
+        //         "deal":"4075612.142786252",
+        //         "change":"1"
+        //     }
+        //
```

Comment:
personally I find it easier to use common symbols and currency pairs such as BTC/USDT, DOGE/USDT, ETH/USDT (equivalently USDC, USDN, USD, if USDT is not present) it just makes it easier to understand what the numbers mean, especially when the response fields have strange names